### DISCUSSION TITLE
https://react-table.js.org/ is broken

### DISCUSSION THREAD

**Discussion by zachrnolan** [TanStack/table#2304] [General] [answered]
<img width="499" alt="Page_Not_Found" src="https://user-images.githubusercontent.com/66614/81733033-38376f00-9446-11ea-9b4b-9a4db787f047.png">

**Comment by ptrobert** [comment]
Hey whats the status on this?

**Comment by tannerlinsley** [comment, marked as answer]
The doc site has been temporarily disabled in favor of the Github docs. The Readme has been reworked to have entry points into the docs, which you can explore via github's UI.
